draft-ietf-quic-http.md

```diff
@@ -2148,6 +2149,9 @@ The entries in {{iana-setting-table}} are registered by this document.
 | ----------------------- | ------- | ----------------------- | --------- |
 {: #iana-setting-table title="Initial HTTP/3 Settings"}
 
+For fomatting reasons, setting names can be abbreviated by removing the
+'SETTING_' prefix.
+
```

Comment:
RFC Editor asked:
> We note that RFC-to-be 9114 registers names in the HTTP/3 Settings registry that are preceded by "SETTINGS_".  They also include the following note in the IANA section: 
> 
>>  For fomatting reasons, the setting names here are abbreviated by 
>> removing the  'SETTING_' prefix.
> 
> Should "MAX_FIELD_SECTION_SIZE" be registered as "SETTINGS_MAX_FIELD_SECTION_SIZE"?  
> See https://www.iana.org/assignments/http3-parameters
> 
> Should a similar note be included in the IANA text as well? 

On re-reading, this may suggest the IANA page itself should be corrected to include SETTINGS_